tests/test_access.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+
+import numpy as np
+import biorbd
+
+from biorbd_optim import (
+    BoundsOption,
+    InterpolationType,
+)
+
+def test_accessors_on_bounds_option():
+    x_min = [-100] * 6
+    x_max = [100] * 6
+    x_bounds = BoundsOption([x_min, x_max], interpolation=InterpolationType.CONSTANT)
+    x_bounds[:3] = 0
```

Comment:
Could you make a second test InterpolationType.CONSTANT_WITH_FIRST.... so you can test multiple columns